.ipynb_checkpoints/hw_2-checkpoint.ipynb

```diff
@@ -0,0 +1,177 @@
+      "text/plain": [
+       "<Figure size 432x288 with 4 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "\n",
+    "def try_ex_func():\n",
```

Comment:
It would be nice if you cared about the user's feelings a little bit more XD
The cons here are: 
- custom text format
- custom parser with "split"
- no default values presumed and hence no fault tolerance at all
The cons here are:
- the user feels agonizing pain instead of experience 
- that's all
From my point of view this is a TRUE GNU_WAY piece of software XD